app/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,8 @@
     <string name="show_as_list">List</string>
     <string name="sorted_by_used_count">Used count</string>
     <string name="sorted_by_used_time">Used time</string>
+    <string name="pair_alerts_desc">App in the background checks the specified exchange rates once a day and if the condition matches, then displays a notification.</string>
+    <string name="ok">OK</string>
+    <string name="privacy_policy_url" translatable="false">https://app-privacy-policy-generator.firebaseapp.com/</string>
```

Comment:
Am I right that we should upload the policy on our website and replace the link here?
Where is the policy itself?

Reply:
Yeah you are right